fossunited/chapters/doctype/foss_chapter/foss_chapter.py

```
@@ -22,11 +22,14 @@ class FOSSChapter(WebsiteGenerator):
 
         about_chapter: DF.TextEditor | None
         banner_image: DF.AttachImage | None
+        bluesky: DF.Data | None
         chapter_logo: DF.AttachImage | None
         chapter_members: DF.Table[FOSSChapterLeadTeamMember]
         chapter_name: DF.Data
-        chapter_status: DF.Literal["Active", "Inactive", "Defunct", "Independent", "New"]  # noqa: F722, F821
-        chapter_type: DF.Literal["City Community", "FOSS Club", "Conference"]  # noqa: F722, F821
+        chapter_status: DF.Literal[
+            "Active", "Inactive", "Defunct", "Independent", "New"  # noqa: F722, F821
```

Ruff (0.13.3)

Unused `noqa` directive (unused: `F722`)

Remove unused `noqa` directive

(RUF100)